fastlane/lib/fastlane/plugins/template/README.md.erb

```diff
@@ -14,6 +14,10 @@ fastlane add_plugin <%= plugin_name %>
 
 <%= summary %>
 
+## Example
+
+Check out the [example `Fastfile`](fastlane/Fastfile) to see how to use this plugin. You can also run it directly from a checkout of this repository using `bundle exec fastlane test`.
```

Comment:
That requires the author of the plugin to set up a sample project, most of the times that's a full iOS/Android project. Maybe we should add more information here? I'd propose:

> Check out the [example `Fastfile`](fastlane/Fastfile) to see how to use this plugin. Try it by cloning the repo, running `fastlane install_plugins` and `bundle exec fastlane test`. 
> **Note to author:** Please set up a sample project to make it easy for users to explore what your plugin does. Provide everything that is necessary to try out the plugin in this project (including a sample Xcode/Android project if necessary)
 
What do you think @neonichu @mfurtak?